scrapers/GirlsOutWest.yml

```diff
@@ -4,20 +4,23 @@ sceneByURL:
     url:
       - tour.girlsoutwest.com/
     scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - tour.girlsoutwest.com/models/
+    scraper: performerScraper
 xPathScrapers:
   sceneScraper:
     scene:
       Title:
         selector: //div[@class='trailerTitle']/div/h3/text()
-        postProcess:
-          - replace:
-            - regex: .+-
-              with:
       Performers:
         Name:
           selector: //div[@class='trailer topSpace']/div//p/a/text()
+        URL:
+          selector: //div[@class='trailer topSpace']/div//p/a/@href
       Image:
-        selector: //div[@class='videoplayer']/img/@src0_1x
+        selector: //div[@class='videoplayer']/img/@src0_3x
```

Comment:
Can you add as a comment the `normal` image also? For some users the 1x might be enough and 3x can be too big sometime